Homework3/Task04/task04.py

```diff
@@ -0,0 +1,5 @@
+def is_armstrong(number: int) -> bool:
+    digits = list(map(int, str(number)))
+    power = len(digits)
+    res = sum(list(map(lambda x: x ** power, digits)))
```

Comment:
why do you need a list here?

Reply:
Same as answered previously.

Reply:
You don't use `len` here.  And see the answer above about sum.